subprojects/docs/src/docs/userguide/authoring-builds/more_about_tasks.adoc

```diff
@@ -928,6 +928,9 @@ The incremental build annotations provide enough information for Gradle to perfo
 * `@InputDirectory` - same as for `@InputFile`, except the path must correspond to a directory.
 * `@OutputDirectory` - verifies that the path doesn’t match a file and also creates the directory if it doesn’t already exist.
 
+If one task produces an output location and another task consumes that location by declaring it as an input, then Gradle validates that the consumer task depends on the consumer task.
```

Comment:
I don't think it's an "output location" that the task produces. Instead, it produces outputs in a location that is then consumed by another task by referring to the location as an input.

I don't think you can use "validates" like that. "Checks" or "confirms" would be a better alternative methinks.

Reply:
I addressed your comments.